search_api_solr.module

```diff
@@ -245,10 +245,10 @@ function search_api_solr_get_data_type_info($type = NULL) {
  */
 function search_api_solr_site_hash() {
   // Copied from apachesolr_site_hash().
-  if (!($hash = config_get('search_api_solr.settings', 'search_api_solr_site_hash'))) {
+  if (!($hash = state_get('search_api_solr.settings', 'search_api_solr_site_hash', FALSE))) {

```

Comment:
state_get takes only two parameters: the name of the state, and the default value. The way you wrote this line, you are storing a string 'search_api_solr_site_hash' under key 'search_api_solr'

Reply:
Oops, I meant to delete the settings part.